programmers/소수-찾기.js

```diff
@@ -0,0 +1,41 @@
+function combine(splitedNums, N) {
+    let result = [];
+    if (N === 1) return splitedNums.map((v) => [v]);
+  
+    splitedNums.forEach((v, idx, splitedNums) => {
+      const fixer = v;
+      const rests = splitedNums.filter((_, index) => index !== idx);
```

Comment:
```suggestion
      const rests = splitedNums.filter((_, index) => index !== idx);
```
filter 안의 " _ " 이건 어떤 의미인가요?

Reply:
filter 메소드는 filter(callback(element, index, array), thisArg)로 구성 되어 있는데, 위에서는 index를 이용하여 필터링하고 있어서 element부분은 필요가 없어요 그래서 그 부분을 공백이나 변수로 처리하는 것 대신에 '_'를 사용하는 것 같습니다.